contentcuration/contentcuration/views/users.py

```diff
@@ -106,10 +106,19 @@ def send_invitation_email(request):
 @api_view(["GET"])
 @authentication_classes((SessionAuthentication,))
 @permission_classes((IsAuthenticated,))
-def deferred_user_data(request):
+def deferred_user_space_by_kind(request):
     return Response(
```

Comment:
Tested using `time.sleep()`, the response of `api_token` isn't affected by how delayed the response of `space_by_kind` is!